setup.cfg

```diff
@@ -1,37 +0,0 @@
-[metadata]
-name = linkml_runtime
-url = https://github.com/biolink/linkml-runtime
-author = Harold Solbrig
-author-email = solbrig@jhu.edu
-summary = LinkML Runtime Environment
-description = Runtime Environment for the Linked Open Data Modeling Language
-home-page = http://linkml.github.io/linkml-runtime
-license = CC0 1.0 Universal
-python-requires = >=3.7
-classifiers =
-    Development Status :: 4 - Beta
```

Comment:
is all this metadata in pyproject.toml

Reply:
Yup, I can see all of this metadata in the pyproject.toml file.